client/state/data-layer/wpcom/me/send-verification-email/test/index.js

```diff
@@ -4,44 +4,85 @@
 import { requestEmailVerification, handleSuccess, handleError } from '../';
 import { EMAIL_VERIFY_REQUEST_SUCCESS, EMAIL_VERIFY_REQUEST_FAILURE } from 'state/action-types';
 import { http } from 'state/data-layer/wpcom-http/actions';
+import { infoNotice, errorNotice, successNotice, removeNotice } from 'state/notices/actions';
+import { translate } from 'i18n-calypso';
 
 describe( 'send-email-verification', () => {
 	describe( '#requestEmailVerification', () => {
-		const dummyAction = { type: 'DUMMY' };
-
 		test( 'should dispatch HTTP request to me/send-verification-email endpoint', () => {
-			expect( requestEmailVerification( dummyAction ) ).toEqual(
+			expect( requestEmailVerification( { type: 'DUMMY' } ) ).toContainEqual(
 				expect.objectContaining(
 					http(
 						{
 							apiVersion: '1.1',
 							method: 'POST',
 							path: '/me/send-verification-email',
 						},
-						dummyAction
+						{ type: 'DUMMY' }
 					)
 				)
 			);
 		} );
+
+		test( 'should dispatch info notice when showing global notices', () => {
+			expect(
+				requestEmailVerification( { type: 'DUMMY', showGlobalNotices: true } )
+			).toContainEqual(
+				infoNotice( translate( 'Sending email…' ), {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Sending email' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/home-janitorial-enhancements)